### PR TITLE
Hello world

### PR DESCRIPTION
**Pull request by dialite**
First project with Microverse titled Hello world

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<link href="index.css" rel="stylesheet">
+<header>
+  <h1>Hello Microverse!</h1>
+</header>
```
Comment (jasonfelice) [inline index.html]: - [ ] Kindly use a proper HTML boilerplate, and make sure to style the header with class. 😄

File: index.css [hunk 2]
```diff
@@ -0,0 +1,3 @@
+header {
+  color: blue;
+}
```
Comment (jasonfelice) [inline index.css]: - [ ] According to the project requirements, you are required to use a class selector.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,97 @@
-# hello-world
-hello world project with microverse
+# HTML & CSS3 Course
```
Comment (jasonfelice) [inline README.md:1]: - [ ] Kindly use [this template](https://github.com/microverseinc/readme-template) for readme.

**Review by jasonfelice — CHANGES_REQUESTED** [review]
Hi @dialite, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
Good commit messages✔️
No linter errors ✔️

## Required Changes ♻️
- [ ] Please add all the linter files.
  image

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.css (hunk 2 above)
Comment (dialite) [reply index.css]: Thank you. I have made the corrections

Comment (dialite) [reply index.css]: Can you check it out again or I will be required to submit it again

Comment (dialite) [reply index.css]: https://github.com/dialite/hello-world/pull/1

**Review by mckent05 — APPROVED** [review]
Hi @dialite, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations and good job! 🎉 

## Optional suggestions
**[OPTIONAL]** - In your future projects, please note that having a more descriptive PR summary will help make your PR more professional. The PR summary could include the changes madein that PR 👍 👍 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by dialite** [comment]
Suggestions received with thanks

**Pull request merged**